src/crypto/CHIPCryptoPAL.cpp

```diff
@@ -21,6 +21,7 @@
  */
 
 #include "CHIPCryptoPAL.h"
+#include "hsm/CHIPCryptoPALHsm.h"
```

Comment:
Protect the header include with #ifdef CHIP_CRYPTO_HSM

Reply:
Sure. Now protected

Reply:
Please review and let me know if the solution is acceptable